### ISSUE TITLE
ConcurrentModificationException

### ISSUE THREAD

**Issue by xyfwdy** [FancyMcPlugins/FancyNpcs#34]
This server is running Folia version git-Folia-"f6e91e9" (MC: 1.20.1) (Implementing API version 1.20.1-R0.1-SNAPSHOT) (Git: f6e91e9)
You are running the latest version
Previous version: git-Folia-"3cf16ee" (MC: 1.20.1)

```
[04:17:36 WARN]: Async task for FancyNpcs v2.0.1 generated an exception
java.util.ConcurrentModificationException: null
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1605) ~[?:?]
	at java.util.HashMap$ValueIterator.next(HashMap.java:1633) ~[?:?]
	at de.oliver.fancynpcs.tracker.NpcTracker.run(NpcTracker.java:24) ~[FancyNpcs-2.0.1.jar:?]
	at de.oliver.fancylib.serverSoftware.schedulers.FoliaScheduler.lambda$runTaskTimerAsynchronously$7(FoliaScheduler.java:60) ~[FancyNpcs-2.0.1.jar:?]
	at io.papermc.paper.threadedregions.scheduler.FoliaAsyncScheduler$AsyncScheduledTask.run(FoliaAsyncScheduler.java:217) ~[folia-1.20.1.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]

```

**Comment by OliverSchlueter** [comment]
Is it fixed in v2.0.1 ?

**Comment by xyfwdy** [comment]
> v2.0.1

It is a v2.0.1 error

**Comment by xyfwdy** [comment]
> Is it fixed in v2.0.1 ?

did u mean the latest build?

**Comment by OliverSchlueter** [comment]
Is it fixed in v2.0.2 ?

**Comment by xyfwdy** [comment]
> Is it fixed in v2.0.2 (https://hangar.papermc.io/Oliver/FancyNpcs/versions/2.0.2) ?
Yes , its work
